physionet-django/console/views.py

```diff
@@ -2985,41 +2986,91 @@ def event_management(request, event_slug):
 
     # handle the add dataset form(s)
     if request.method == "POST":
-        if 'add-event-dataset' in request.POST.keys():
+        if "add-event-dataset" in request.POST.keys():
             event_dataset_form = EventDatasetForm(request.POST)
             if event_dataset_form.is_valid():
-                if selected_event.datasets.filter(
-                        dataset=event_dataset_form.cleaned_data['dataset'],
-                        access_type=event_dataset_form.cleaned_data['access_type'],
-                        is_active=True).count() == 0:
+                if (
+                    selected_event.datasets.filter(
+                        dataset=event_dataset_form.cleaned_data["dataset"],
+                        access_type=event_dataset_form.cleaned_data["access_type"],
+                        is_active=True,
+                    ).count()
+                    == 0
+                ):
```

Comment:
I'd refrain from huge logic like this inside the if statement. I'd rather see a meaningful variable name here, like `active_datasets` or something.